src/routes/(app)/contacts/graph.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import Icon from '@/lib/components/icons/icon.svelte';
+  import Icons from '$lib/components/icons/icons';
+  import { copyToClipboard } from '$lib/utils';
+
+  const coords = [
+    [1, 1],
+    [6, 3],
+    [2, 5],
+    [10, 6],
+    [1, 8],
+    [7, 8]
+  ];
+  const lines = [
+    [0, 2],
+    [2, 1],
+    [1, 5],
+    [5, 3],
+    [2, 4]
+  ];
+  let innerWidth = 0;
+  let coefficient = 8;
+  let iconSize = 8;
+
+  const socials = [
+    { url: 'https://www.instagram.com/niaefeup/', icon: Icons.Instagram, label: 'Instagram' },
+    { url: 'https://github.com/NIAEFEUP', icon: Icons.Github, label: 'Github' },
+    { url: 'https://www.facebook.com/NIAEFEUP', icon: Icons.Facebook, label: 'Facebook' },
+    { url: 'ni@aefeup.pt', icon: Icons.Mail, label: 'Email' },
+    { url: 'https://www.linkedin.com/company/nifeup', icon: Icons.Linkedin, label: 'Linkedin' },
+    { url: 'https://twitter.com/niaefeup', icon: Icons.Twitter, label: 'Twitter' }
+  ];
+</script>
+
+<svelte:window bind:innerWidth />
+
+<svg style="height: 40dvh; min-width: 35dvw" viewBox="0 0 87 80" xmlns="http://www.w3.org/2000/svg">
+  <!-- Draw graph edges. -->
+  {#each lines as line}
+    <line
+      x1={coords[line[0]][0] * coefficient}
+      y1={coords[line[0]][1] * coefficient}
+      x2={coords[line[1]][0] * coefficient}
+      y2={coords[line[1]][1] * coefficient}
+      class="stroke-vivid-red-900"
+    />
+  {/each}
+
+  <!-- Draw graph nodes. -->
+  {#each coords as coord, index}
+    <foreignObject
+      x={coord[0] * coefficient - (iconSize + 2) / 2}
+      y={coord[1] * coefficient - (iconSize + 2) / 2}
+      width={iconSize + 2}
+      height={iconSize + 2}
+    >
+      {#if socials[index].url.startsWith('https')}
+        <div class="flex h-full w-full items-center justify-center rounded bg-white bg-opacity-30">
+          <Icon
+            src={socials[index].icon}
+            color="white"
+            size="{iconSize.toString()}px"
+            href={socials[index].url}
+            ariaLabel={socials[index].label}
+          />
+        </div>
+      {:else}
+        <div
+          class="flex h-full w-full items-center justify-center rounded bg-white bg-opacity-30"
+          on:click={() => copyToClipboard(socials[index].url)}
+          on:keydown={() => copyToClipboard(socials[index].url)}
+          role="button"
+          tabindex="0"
+          aria-label="copy-mail"
+        >
+          <Icon src={socials[index].icon} color="white" size="{iconSize.toString()}px" />
+        </div>
+      {/if}
+    </foreignObject>
+  {/each}
+</svg>
```

Comment:
Although less reusable it'd be nice to have this on storybook too, for example to test responsivity easily :) And please place this in `src/routes/(app)/contacts/_components` for consistency